packages/website/ts/components/portal/portal.tsx

```diff
@@ -241,6 +271,141 @@ export class Portal extends React.Component<PortalProps, PortalState> {
             </div>
         );
     }
+    private _renderMenu(routeComponentProps: RouteComponentProps<any>): React.ReactNode {
+        return (
+            <div>
+                <BackButton to={`${WebsitePaths.Portal}`} labelText={'back to Relayers'} />
```

Comment:
Don't need `{}`

Reply:
I think in this case he does because it's a template literal? Could be wrong.